onboard-server/src/main/java/no/vegvesen/ixn/serviceprovider/OnboardRestController.java

```diff
@@ -184,10 +172,7 @@ public void deleteSubscription(@PathVariable String serviceProviderName, @PathVa
 
 
 
-		ServiceProvider serviceProviderToUpdate = serviceProviderRepository.findByName(serviceProviderName);
-		if (serviceProviderToUpdate == null) {
-			throw new NotFoundException("The Service Provider trying to delete a subscription does not exist in the database. No subscriptions to delete.");
```

Comment:
Same as for deleting Capability, cast an NotFoundException when the requested Ssubcription for delete is not found. And should we create a ServiceProvider here

Reply:
As abobve, when we try to delete a capability we cannot find, we throw NotFoundException